tests/integration/test_prometheus_protocols/test.py

```diff
@@ -202,3 +225,16 @@ def get(path):
     assert "DB::Exception: snappy uncomress failed" in auth_ok.text
 
     assert get("/read_auth_fail").status_code == 403
+
+
+def test_remote_write_v1_status_code():
+    node.query("CREATE TABLE prometheus ENGINE=TimeSeries")
+    body = bytes.fromhex(
+        "0a380a100a085f5f6e616d655f5f120474696d650a130a06726567696f6e120975732d656173742d31120f09000000000080404010afdcb5bc061a1a0802120474696d65220c48656c70204d6573736167652a026d73"
```

Comment:
@bluestealth Would be nice to add a comment how did you get this data.

Reply:
Good point, I can add the object I used to create this in golang, unfortunately python client does not implement remote-write or protobuf types. Hence why I dumped the encoded object. Although the python implementations could be built from sources using protoc, it seemed kind of heavy for just the one.

Reply:
@nikitamikhaylov added the source I used to generate this